### ISSUE TITLE
Regression in "sticky comments" commit -- extra newline inserted

### ISSUE THREAD

**Issue by timgraham** [timothycrosley/isort#238]
A failing test (passes before de68bee9b875ba7e7eb2b614b4bc786f95f8b7f1):
```
def test_top_comments():
    test_input = (
        "# -*- encoding: utf-8 -*-\n"
        "# Test comment\n"
        "#\n"
        "from __future__ import unicode_literals\n"
    )
    assert SortImports(file_contents=test_input).output == test_input
```
After de68bee9b875ba7e7eb2b614b4bc786f95f8b7f1, isort inserts an extra blank line like this:
```
# -*- encoding: utf-8 -*-
# Test comment

#
from __future__ import unicode_literals
```

**Comment by timothycrosley** [comment]
Hi @timgraham,

Good catch! I'll see what I can do to work around this issue.

Thanks! 

~Tim

**Comment by timgraham** [comment]
Fixed by bd8b38542d5645762f1d0ecdde4f34da6b32ef02 and e1b2f3cd9e59ee006c638df182feb7850f625dfe
